src/recommendations/implementation.md

```diff
@@ -0,0 +1,41 @@
+---
+group: product-recommendations
+title: How to Implement Product Recommendations
+ee_only: True
+---
+
+To help you implement product recommendations on your storefront, use the following workflow:
+
+1. **Deploy data collection to production**
+
+   Product Recommendations are made up of two main data sources: [catalog and behavioral]({{ page.baseurl }}/recommendations/product-recs.html#types-of-data). By deploying data collection to production, you are enabling a JavaScript application to collect the behavioral data. The more data you collect, the more accurate your recommendations will be. As soon as you start collecting data, you can pull that data into your non-production environment and experiment with different recommendation types on different pages.
```

Comment:
```suggestion
   Deploying Product Recommendations requires two main data sources: [catalog and behavioral]({{ page.baseurl }}/recommendations/product-recs.html#types-of-data). Because production is the only environment where your shoppers' actions can captured and analyzed, it's in your best interest to start data collection on production as early as possible. The more data you collect by the time you start showing recommendations to shoppers, the higher quality you can expect your recommendations to be. As an added benefit, when you start collecting behavioral data on production, you can fetch Recommendations based on this production data while operating in non-production environments [ADD LINK]. This enables the user to test and experiment with different recommendations that are computed based on real shopper data collected in production.
```